mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -95,6 +95,21 @@ def run(self):
         ip = n.allocate_ip()
         spec['ip'] = ip
 
+        # Get 'mizar.com/egress-bandwidth' from pod annotations
+        egress_bw_bps = int(0)
+        annotations = self.param.body['metadata'].get('annotations', {})
+        if len(annotations) > 0:
+            k8s_egress_bw_bps = annotations.get(CONSTANTS.MIZAR_EGRESS_BW_TAG)
+            if k8s_egress_bw_bps is not None:
+                egress_bw_bps = k8s_egress_bw_bps
+                if k8s_egress_bw_bps.endswith('K'):
+                    egress_bw_bps = int(float(k8s_egress_bw_bps.replace('K', '')) * 1e3)
```

Comment:
Should this be 1e3 or 1024?

Reply:
@Hong-Chang It looks like the current conversion is correct besides the need to multiply by 8 to convert it to bits per second (bps).  See https://en.wikipedia.org/wiki/Kilobyte
In kubernetes, the annotation such as "kubernetes.io/egress-bandwidth": "10M" refers to 10M/s (Megabytes per second = 1000x1000 bytes per second). The binary notation (Mi/s) would imply 1024x1024.

Reply:
I see. Thanks.